telegram/_files/file.py

```diff
@@ -58,18 +58,19 @@ class File(TelegramObject):
         file_unique_id (:obj:`str`): Unique identifier for this file, which
             is supposed to be the same over time and for different bots.
             Can't be used to download or reuse the file.
-        file_size (:obj:`int`, optional): Optional. File size in bytes, if known.
+        file_size (:obj:`int`, optional): File size in bytes, if known.
         file_path (:obj:`str`, optional): File path. Use e.g. :meth:`download_to_drive` to get the
             file.
 
     Attributes:
-        file_id (:obj:`str`): Identifier for this file.
+        file_id (:obj:`str`): Identifier for this file, which can be used to download
+            or reuse the file.
         file_unique_id (:obj:`str`): Unique identifier for this file, which
             is supposed to be the same over time and for different bots.
             Can't be used to download or reuse the file.
-        file_size (:obj:`str`): Optional. File size in bytes.
-        file_path (:obj:`str`): Optional. File path. Use e.g. :meth:`download_to_drive` to get
-            the file.
+        file_size (:obj:`int`): Optional. File size in bytes, if known.
```

Comment:
[Telegram API](https://core.telegram.org/bots/api#file) has following addition to `file_size` attr. Since it contains the phrase "some programming languages", we may not need to include it.

> It can be bigger than 2^31 and some programming languages may have difficulty/silent defects in interpreting it. But it has at most 52 significant bits, so a signed 64-bit integer or double-precision float type are safe for storing ths value.

Reply:
yeah I vote not to include it